Pod/Classes/WPMediaCollectionViewCell.m

```diff
@@ -314,22 +320,31 @@ - (void)setPlaceholderTintColor:(UIColor *)placeholderTintColor
 - (void)setSelected:(BOOL)selected
 {
     [super setSelected:selected];
-    if (self.isSelected) {
-    } else {
-        self.positionLabel.hidden = YES;
-    }
+    [self updatePositionLabelToSelectedState:self.isSelected];
 }
 
 - (void)tintColorDidChange
 {
     [super tintColorDidChange];
     _selectionFrame.layer.borderColor = [[self tintColor] CGColor];
-    _positionLabel.backgroundColor = [self tintColor];
-    if (self.isSelected) {
 
+    [self updatePositionLabelToSelectedState:self.isSelected];
+}
+
+- (void)updatePositionLabelToSelectedState:(BOOL)selected
+{
+    if (selected) {
+        _positionLabel.backgroundColor = [self tintColor];
+        _positionLabel.layer.borderColor = [self tintColor].CGColor;
+        _positionLabel.layer.shadowColor = [UIColor colorWithWhite:0 alpha:1].CGColor;
+        _positionLabel.layer.shadowRadius = 10.0;
+        _positionLabel.layer.shadowOffset = CGSizeMake(10, 10);
     } else {
-        
+        _positionLabel.text = @"";
+        _positionLabel.backgroundColor = [[UIColor lightGrayColor] colorWithAlphaComponent:0.7];
```

Comment:
Good catch, I think I'm going to make this color configurable trough an appearance method to make it easier to configure for other users of the library.